content/surveys/_index.en.md

```diff
@@ -5,35 +5,94 @@ tags: ["wiki"]
+
+The higher the DEFF, the higher the variability between clusters and the higher the probability that subjects within a cluster are similar.
+
+In a well designed survey, you will aim to have a calculated DEFF for your outcome of interest as close as possible to 1. This is because a DEFF of 1 means that your survey design has approached Simple Random Sampling (SRS) for your outcome of interest. SRS is the gold standard in survey design, but often not possible in MSF contexts. In general, in two-stage cluster sampling surveys, we would assume a DEFF of 2.
+
+Mathematically DEFF is calculated as:
+
+> **DEFF=1+(1-n) * rho**
+> where n=sample size of your survey and rho=intraclass coefficient
+
+### Precision around your estimate
+The precision around your estimate stipulates how close to the your estimate you would like the 95% confidence interval of your prevalence estimate to fall.
+
+> **For example**: you have calculated a 72% measles vaccination coverage in your study population. You would like to ensure that the 
+> 95% confidence interval is narrow (between 70-74%); thus you would set your precision at 2% (2% above and below 72%).
+
+The lower your precision, the higher your sample size calculation will be.
+
+## Data collection and databases
+The templates assume that data will have been collected using mobile data collection or will have been entered into an electronic database (Excel, Epi Data, Redcap etc.).
+
+We recommend that you try and establish generic data dictionaries for mobile data collection (through template questionnaires) or through template databases. This will improve the consistency of the naming of your variables and will facilitate the frequent and systematic use of the Survey Templates.
+
+## Survey analysis, the basics
+In the Survey Template Walkthrough (INSERT LINK), there is a step by step description of how to use the templates for data analysis.
+
+We have designed the templates to run through a similar series of analytical questions in the same order:
+
+1. **Description of your survey** sample and population in that survey sample
+2. **Comparison of your survey sample** to the population breakdown of that area (this will help you understand how biased your data is)
+3. **Calculation of the main outcomes of interest** (including DEFF): 
+
+  * Mortality survey template: crude mortality rate (CMR) and under 5 year mortality rate (U5MR)
+  * Vaccination coverage: vaccination coverage of vaccines of interest
+  * Malnutrition: prevalence of malnutrition in target population through MUAC and Weight for Height Scores  
+  
+4. Calculation of **secondary outcomes of interest**
+
+The sampling stratgey **MUST** be considered by the epidemiologist using the templates in order to ensure an appropriate weighting is applied to the analysis.
```

Comment:
Adding a notice box to strengthen the statement here. https://learn.netlify.com/en/shortcodes/notice/

```suggestion
{{% notice info %}}
The sampling stratgey **MUST** be considered by the epidemiologist using the templates in order to ensure an appropriate weighting is applied to the analysis.
{% /notice %}}
```